.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = externals/fmt
 	url = https://github.com/fmtlib/fmt.git
 
+[submodule "externals/faker-cxx"]
```

Comment:
I am not sure of this change, should I remove this?

Reply:
Yes, please remove